API/test/integration/integration.go

```diff
@@ -32,8 +32,9 @@ func recreateTestDB() {
 	ctx, _ := utils.Connect()
 
 	err = db.Drop(ctx)
-	if err != nil {
-		log.Fatalln("Error while doing drop:", err.Error())
+	for err != nil {
+		log.Println("Error while doing drop:", err.Error())
+		err = db.Drop(ctx)
```

Comment:
drop again and not check the return?

Reply:
It's not necessarily the second one that's going to work, that's why it's within a for, basically keep trying until it works. Anyway sometimes it keeps breaking, I'll look for a better solution tomorrow